jrnl/override.py

```diff
@@ -0,0 +1,47 @@
+# import logging
+def apply_overrides(overrides: list, base_config: dict) -> dict:
```

Comment:
There is a similar `update_config` method in config.py that merges configuration of individual journals into the global config. Is there a reason to keep it separate?

Reply:
I'd only ever seen it used in the context of applying the --encrypt and --decrypt postconfig commands, but I don't see any argument against just calling update_config after unpacking . Lemme work on updating the implementation.